LeftNavBarExample/LeftNavBarLibrary/build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.0.0'
+        classpath 'com.android.tools.build:gradle:2.2.1'
```

Comment:
My considerations comes from the realisation every module is an app on its own. I see the plus of the simpler import, but that means also longer evaluation times (the whole root project is evaluated in the process 😅 ). I'm not saying you should change the rest of the project, but I would seriously consider not to follow the (broken IMHO) pattern and make this a separate project on its own (all you need is avoid to list your module in the root `settings.gradle`, right?).